tests/test_threads.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+from confluent_kafka import Producer, KafkaError, KafkaException
+import threading, time, queue
```

Comment:
Hmm, `queue` is the py3 name, whereas `Queue` is used in py2. Currently this fails on py2 for me.